views/default/forms/usersettings/save.php

```diff
@@ -5,10 +5,18 @@
  * Plugins should extend "forms/account/settings" to add to the settings.
  */
 
-$form_body = elgg_view("forms/account/settings", $vars);
+$user = elgg_get_page_owner_entity();
 
-$form_body .= '<div class="elgg-foot">';
-$form_body .= elgg_view('input/submit', array('value' => elgg_echo('save')));
-$form_body .= '</div>';
+$form_body = elgg_view('forms/account/settings', $vars);
+
+$submit = elgg_view('input/submit', ['value' => elgg_echo('save')]);
+$hidden = '';
```

Comment:
if we do a `if ($user) {`  we need to do the `$hidden = '';` if we find it logical to have the user always available when using this form, why not do a `if (!user) return;` as the first check.

I think for now this should stay the way it is